sklearn/linear_model/sgd_fast.pyx

```diff
@@ -319,7 +319,7 @@ cdef class SquaredEpsilonInsensitive(Regression):
         z = y - p
         if z > self.epsilon:
             return -2 * (z - self.epsilon)
-        elif z < self.epsilon:
+        elif z < -self.epsilon:
```

Comment:
Hi Alexandre, I will close this PR and start a new PR include commits of what's new document and this file, is that better, or just start a new PR only include what's new document.